### PR TITLE
Make some definitions lower case that are not

### PR DESCRIPTION
**Pull request by matentzn**
This makes a few hundred definitions upper case that were previously lower case.

### CODE REVIEW

File: src/ontology/mondo-edit.obo [hunk 1]
```diff
@@ -107747,7 +107747,7 @@ relationship: disease_has_feature HP:0010762 ! Chordoma
 [Term]
 id: MONDO:0006146
 name: chondroid hamartoma
-def: "https://ncit.nci.nih.gov/ncitbrowser/ConceptReport.jsp?dictionary=NCI_Thesaurus&code=C42589&ns=NCI_Thesaurus" [EFO:1000175]
+def: "Https://ncit.nci.nih.gov/ncitbrowser/ConceptReport.jsp?dictionary=NCI_Thesaurus&code=C42589&ns=NCI_Thesaurus" [EFO:1000175]
```
Comment (nicolevasilevsky) [inline src/ontology/mondo-edit.obo:107750]: I don't think we should have hyperlinks in the def field, I imagine this is a mistake

File: src/ontology/mondo-edit.obo [hunk 2]
```diff
@@ -110720,7 +110720,7 @@ intersection_of: disease_has_location UBERON:0001707 ! nasal cavity
 [Term]
 id: MONDO:0006315
 name: neoplastic medium-sized B-lymphocyte with basophilic cytoplasm
-def: "https://ncit.nci.nih.gov/ncitbrowser/ConceptReport.jsp?dictionary=NCI_Thesaurus&code=C37005&ns=NCI_Thesaurus" [EFO:1000392]
+def: "Https://ncit.nci.nih.gov/ncitbrowser/ConceptReport.jsp?dictionary=NCI_Thesaurus&code=C37005&ns=NCI_Thesaurus" [EFO:1000392]
```
Comment (nicolevasilevsky) [inline src/ontology/mondo-edit.obo:110723]: also here - probably a mistake

File: src/ontology/mondo-edit.obo [hunk 3]
```diff
@@ -110855,7 +110855,7 @@ relationship: excluded_subClassOf MONDO:0006320 {source="NCIT:C8562", source="ht
 [Term]
 id: MONDO:0006326
 name: ocular melanoma with extraocular extension
-def: "https://ncit.nci.nih.gov/ncitbrowser/ConceptReport.jsp?dictionary=NCI_Thesaurus&code=C7913&ns=NCI_Thesaurus" [EFO:1000404]
+def: "Https://ncit.nci.nih.gov/ncitbrowser/ConceptReport.jsp?dictionary=NCI_Thesaurus&code=C7913&ns=NCI_Thesaurus" [EFO:1000404]
```
Comment (nicolevasilevsky) [inline src/ontology/mondo-edit.obo:110858]: looks like there are a lot of these

File: src/ontology/mondo-edit.obo [hunk 4]
```diff
@@ -130381,7 +130381,7 @@ relationship: disease_has_basis_in_dysfunction_of http://identifiers.org/hgnc/42
 [Term]
 id: MONDO:0007307
 name: Charcot-Marie-Tooth disease type 1B
-def: "age 40), with normal or mildly slowed MNCV and more frequent hearing loss and pupillary abnormalities. CMT1B can also cause the classical CMT phenotype in about 15% of total CMT1B cases." [Orphanet:101082]
+def: "Age 40), with normal or mildly slowed MNCV and more frequent hearing loss and pupillary abnormalities. CMT1B can also cause the classical CMT phenotype in about 15% of total CMT1B cases." [Orphanet:101082]
```
Comment (nicolevasilevsky) [inline src/ontology/mondo-edit.obo:130384]: this def looks like it is cut off

File: src/ontology/mondo-edit.obo [hunk 5]
```diff
@@ -148914,7 +148914,7 @@ property_value: seeAlso https://rarediseases.info.nih.gov/diseases/3818/multicen
 [Term]
 id: MONDO:0008153
 name: progressive osseous heteroplasia
-def: "gene (20q13.2-q13.3), coding for guanine nucleotide-binding proteins. POH can, however, be distinguished clinically by the deep and progressive nature of the heterotopic bone formation." [Orphanet:2762]
+def: "Gene (20q13.2-q13.3), coding for guanine nucleotide-binding proteins. POH can, however, be distinguished clinically by the deep and progressive nature of the heterotopic bone formation." [Orphanet:2762]
```
Comment (nicolevasilevsky) [inline src/ontology/mondo-edit.obo:148917]: this def looks like it is cut off

File: src/ontology/mondo-edit.obo [hunk 6]
```diff
@@ -158235,7 +158235,7 @@ relationship: disease_has_basis_in_dysfunction_of http://identifiers.org/hgnc/36
 [Term]
 id: MONDO:0008594
 name: familial multiple discoid fibromas
-def: "gene locus mutation." [Orphanet:538756]
+def: "Gene locus mutation." [Orphanet:538756]
```
Comment (nicolevasilevsky) [inline src/ontology/mondo-edit.obo:158238]: this def looks like it is cut off

File: src/ontology/mondo-edit.obo [hunk 7]
```diff
@@ -164276,7 +164276,7 @@ replaced_by: MONDO:0005260
 [Term]
 id: MONDO:0008852
 name: congenital central hypoventilation syndrome
-def: "gene is found in 90% of the patients. Association with a Hirschsprung's disease is observed in 16% of the cases. Despite a high mortality rate and a lifelong dependence to mechanical ventilation, the long-term outcome of CCHS should be ultimately improved by multidisciplinary and coordinated follow-up of the patients." [Orphanet:661]
+def: "Gene is found in 90% of the patients. Association with a Hirschsprung's disease is observed in 16% of the cases. Despite a high mortality rate and a lifelong dependence to mechanical ventilation, the long-term outcome of CCHS should be ultimately improved by multidisciplinary and coordinated follow-up of the patients." [Orphanet:661]
```
Comment (nicolevasilevsky) [inline src/ontology/mondo-edit.obo:164279]: I think a lot of these were cut off, hence they were lowercase

File: src/ontology/mondo-edit.obo [hunk 8]
```diff
@@ -167275,7 +167275,7 @@ property_value: confidence "8.333333333333334" xsd:double
 [Term]
 id: MONDO:0008974
 name: Greenberg dysplasia
-def: "lethality and affected fetuses are considered as nonviable." [Orphanet:1426]
+def: "Lethality and affected fetuses are considered as nonviable." [Orphanet:1426]
```
Comment (nicolevasilevsky) [inline src/ontology/mondo-edit.obo:167278]: cut off probably

File: src/ontology/mondo-edit.obo [hunk 9]
```diff
@@ -169451,7 +169451,7 @@ property_value: confidence "1.3333333333333335" xsd:double
 [Term]
 id: MONDO:0009070
 name: D-glyceric aciduria
-def: "gene has been mapped to 3p21." [Orphanet:941]
+def: "Gene has been mapped to 3p21." [Orphanet:941]
```
Comment (nicolevasilevsky) [inline src/ontology/mondo-edit.obo:169454]: cut off probably

File: src/ontology/mondo-edit.obo [hunk 10]
```diff
@@ -182116,7 +182116,7 @@ is_a: MONDO:0003847 {source="MESH:C535875/inferred"} ! Mendelian disease
 [Term]
 id: MONDO:0009603
 name: 3-hydroxyisobutyryl-CoA hydrolase deficiency
-def: "gene, encoding 3-hydroxyisobutyryl-CoA hydrolase. The mode of transmission has not yet been established." [Orphanet:88639]
+def: "Gene, encoding 3-hydroxyisobutyryl-CoA hydrolase. The mode of transmission has not yet been established." [Orphanet:88639]
```
Comment (nicolevasilevsky) [inline src/ontology/mondo-edit.obo:182119]: cut off probably

File: src/ontology/mondo-edit.obo [hunk 11]
```diff
@@ -187398,7 +187398,7 @@ property_value: seeAlso https://rarediseases.info.nih.gov/diseases/8662/osteodys
 [Term]
 id: MONDO:0009803
 name: congenital osteogenesis imperfecta-microcephaly-cataracts syndrome
-def: "or a few hours after birth. The mode of inheritance appears to be autosomal recessive." [Orphanet:2772]
+def: "Or a few hours after birth. The mode of inheritance appears to be autosomal recessive." [Orphanet:2772]
```
Comment (nicolevasilevsky) [inline src/ontology/mondo-edit.obo:187401]: ha, this has to be cut off

File: src/ontology/mondo-edit.obo [hunk 12]
```diff
@@ -189427,7 +189427,7 @@ property_value: confidence "2.291666666666666" xsd:double
 [Term]
 id: MONDO:0009880
 name: short stature-pituitary and cerebellar defects-small sella turcica syndrome
-def: "gene (1q25)." [Orphanet:85442]
+def: "Gene (1q25)." [Orphanet:85442]
```
Comment (nicolevasilevsky) [inline src/ontology/mondo-edit.obo:189430]: cut off

File: src/ontology/mondo-edit.obo [hunk 13]
```diff
@@ -189407,7 +189407,7 @@ property_value: confidence "1.1441176470588243" xsd:double
 [Term]
 id: MONDO:0009879
 name: short stature due to growth hormone qualitative anomaly
-def: "gene (17q22-q24) that result in structural GH anomalies and a biologically inactive molecule. Transmission is autosomal recessive." [Orphanet:629]
+def: "Gene (17q22-q24) that result in structural GH anomalies and a biologically inactive molecule. Transmission is autosomal recessive." [Orphanet:629]
```
Comment (nicolevasilevsky) [inline src/ontology/mondo-edit.obo:189410]: cut off

**Review by nicolevasilevsky — CHANGES_REQUESTED** [review]
I think the reason why a lot of these defs were lowercase is because they were cut off - I think this happened with some Orphanet terms.

Not sure the best approach to fix this - could we re-import the definitions?

Or I can manually fix.

**Comment by matentzn** [comment]
Ok, lets close this and try to work out a way to do this right; see #1979

**Comment by matentzn** [comment]
Lets try this again another time.. :)

**Pull request closed without merging**